addons/website/static/tests/tours/edit_translated_page.js

```diff
@@ -0,0 +1,19 @@
+/** @odoo-module **/
+
+import tour from "web_tour.tour";
+
+tour.register('edit_translated_page_redirect', {
+    test: true,
+    url: '/nl_NL',
```

Comment:
```suggestion
    url: '/nl/contactus',
```
as you defined in your python